master.js

```diff
@@ -0,0 +1,151 @@
+        let arr = [id, description, userID, status];
+        for(let i=0; i<4; i++){
+            let td = document.createElement("td");
+            let text = document.createTextNode(arr[i])
+            td.appendChild(text);
+            if(arr[i] == description){
+                td.classList.add("description")
+                td.setAttribute("title", description)
+            }
+            tableRow.appendChild(td)
+        }
+        let deleteBtn = document.createElement("button");
+        deleteBtn.id = "delete"
+        deleteBtn.dataset.id = obj.id;
+        deleteBtn.appendChild(document.createTextNode("Delete"))
+        let doneBtn = document.createElement("button");
+        doneBtn.id = "done"
+        doneBtn.dataset.id = obj.id;
+        if(obj.completed)
+            doneBtn.appendChild(document.createTextNode("Undo"))
+        else doneBtn.appendChild(document.createTextNode("Done")) 
+        let td = document.createElement("td");
+        td.append(deleteBtn, doneBtn)
+        tableRow.appendChild(td)
+        obj.completed ? tableRow.style.background = "#cccccc3d": null
+        tbody.appendChild(tableRow)
+    })
+    let count = document.querySelector(".count span")
+    count.innerHTML = filterdData.length;}
+}
+
+//**************************************************
+function clearTable(){
+    let tbody = document.querySelector("table tbody")
+    tbody.innerHTML = ""
+}
+function done(){
+    document.addEventListener("click",(e)=>{
+        if(e.target.id == "done"){
+            let rowId = e.target.dataset.id;
+            data.forEach((obj)=>{
+                if (obj.id == rowId){
+                    obj.completed ? obj.completed = false : obj.completed = true;
+                }
+            })
+            window.localStorage.setItem("list",JSON.stringify(data))
+            clearTable();
+            fillTable(data)
+        }
+    })
+}
+done()
+
+function deleteTask(){
+    document.addEventListener("click",(e)=>{
+        if(e.target.id == "delete"){
+            Swal.fire({
+                title: 'Cation!',
+                text: 'Are you sure you want continue delete this todo ?',
+                icon: 'warning',
+                footer: "Note: you cant undo this operation",
+                showDenyButton:true,
+                denyButtonText:"No",
+                confirmButtonText:"yes sure"
+              }).then((prom)=>{
+                if(prom.isConfirmed == true){
+                    let rowId = e.target.dataset.id;
+                    data.forEach((obj)=>{
+                        if (obj.id == rowId){
+                            let indexToRemove = data.indexOf(obj)
+                            data.splice(indexToRemove,1)
+                        }
+                    })
+                    window.localStorage.setItem("list",JSON.stringify(data))
+                    clearTable();
+                    fillTable(data)
+                }
+              })
+        }
```

Comment:
I would recommend to implement this from scratch instead of using the ready ones